### DISCUSSION TITLE
卡在了提交表单这一步

### DISCUSSION THREAD

**Discussion by comeonsunny** [MasterKenway/DGUT-yqfk#9] [Q&A]
![image](https://user-images.githubusercontent.com/84911039/135615576-836d9a37-460b-42e8-98e2-2d852646f517.png)
在函数post-form里面，最后的results的值：选定的 核酸检测结果 是无效的
这也不知道问题处在了那里？求大神指教

**Comment by MasterKenway** [comment]
> ![image](https://user-images.githubusercontent.com/84911039/135615576-836d9a37-460b-42e8-98e2-2d852646f517.png)
> 
> 在函数post-form里面，最后的results的值：选定的 核酸检测结果 是无效的
> 
> 这也不知道问题处在了那里？求大神指教

脚本会获取上一次填写的表单，之前是否填了这个字段？
